### PR TITLE
Add a from me banner and fix grunt watch

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
- As we are an open source project I think we should link to our open source repo on GitHub, and what better way then using the banners provided by them.
- We have discovered the Watch task is broken from another PR, this is also fixed in this branch.

#### What tests does this PR have?
None

#### How can be this tested?
None

#### Screenshots / Screencast
screen shot 2015-06-01 at 12 42 39

#### What gif best describes how you feel about this work?
collapse

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the contributing document and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve run all the tests locally and they pass.

#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve run all the tests locally and they pass.

### CODE REVIEW

File: grunt/watch.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 module.exports = {
-	docs: {
-		files: ['docs/**/*.jsx', 'docs/**/*.js', 'src/**/*.js*'],
-		tasks: ['docs-build'],
-		options: {
-			livereload: true
-		}
-	},
+  docs: {
```
Comment (lukefrake) [inline grunt/watch.js]: Indentation has gone a bit off the rails here

File: grunt/watch.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+      files: ['docs/src/**/*.js*', 'src/**/*.js*'],
+      // files: ['docs/**/*.jsx', 'src/**/*.jsx', 'src/**/*.js'],
```
Comment (lukefrake) [inline grunt/watch.js]: Any reason to keep this?

File: docs/index.html [hunk 3]
```diff
@@ -61,6 +61,7 @@
   </head>
   <body>
     <div id="application"></div>
+    <a href="https://github.com/holidayextras/ui-toolkit"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
```
Comment (lukefrake) [inline docs/index.html]: Any reason to keep this inline?

File: grunt/shell.js [hunk 4]
```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  ulimit: {
+    command: "ulimit -n 9999",
```
Comment (lukefrake) [inline grunt/shell.js]: Don't need a , at the end of this

File: docs/index.html (hunk 3 above)
Comment (jodiedoubleday) [inline docs/index.html]: Comes direct from GitHub badges page.
I could copy them over to UIToolkit Less but don't see too much need.

Comment (lukefrake) [inline docs/index.html]: Sounds cool, thought it would be something that makes sense :+1:

**Comment by lukefrake** [comment]
:+1:

**Comment by hx-markterry** [comment]
taken

**Comment by hx-markterry** [comment]
still get the following ulimit related error on my machine:

http://grab.by/HJyg

**Comment by hx-markterry** [comment]
if you change the `docs/package.json` to:

    docsBuild: {
      command: 'cd docs; npm run build && cd -'
    },
    docsStart: {
      command: 'cd docs; npm start && cd -'
    },

Then if the docs fail the error will bubble up to the main grunt task

File: grunt/shell.js [hunk 5]
```diff
@@ -1,14 +1,17 @@
 module.exports = {
+  ulimit: {
+    command: 'ulimit -n 9999'
```
Comment (hx-markterry) [inline grunt/shell.js]: This won't do what you want. The ulimit command is used to set limit for the current shell and any spawned processes. The grunt-shell plugin spawns a new shell to run its tasks so this command will only affect commands in the same shell not other grunt tasks.

To resolve the issue i would remove this task and add the ulimit statement to the top of the problem bash script at `docs/script/build.sh` fixing it at the source.

File: grunt/shell.js [hunk 6]
```diff
@@ -1,14 +1,17 @@
...
+  },
   buildDist: {
     command: "./node_modules/.bin/browserify -g [ ./node_modules/reactify --extension '.jsx' ] ./index.js --standalone ui-toolkit > ./dist/ui-toolkit.standalone.js"
   },
   docsBuild: {
-    command: 'cd docs; npm run build; cd -',
+    command: 'cd docs; npm run build; cd -'
```
Comment (hx-markterry) [inline grunt/shell.js]: Change to
    command: 'cd docs; npm run build && cd -'

To allow any errors from the build command to bubble up

File: grunt/shell.js [hunk 7]
```diff
@@ -1,14 +1,17 @@
...
   },
   docsStart: {
-    command: 'cd docs; npm start; cd -',
+    command: 'cd docs; npm start; cd -'
```
Comment (hx-markterry) [inline grunt/shell.js]: Change to

    command: 'cd docs; npm run start && cd -'

To allow any errors from the start command to bubble up

**Comment by hx-markterry** [comment]
this works for me now, i still get the initial server not found error, but this is due to the excessive separation of grunt tasks in the docs dir. This can be resolved another time.

**Pull request merged**